Tests/GPU/RandomNumberGeneration/main.H

```diff
@@ -0,0 +1,15 @@
+#ifndef MAIN_H
```

Comment:
I think this header file isn't needed any more.

Reply:
Should I include AMReX.H and the other headers in the main,cpp file then?

Reply:
Yes, I would just include them directly.

Reply:
Will do.

Reply:
modified the main.cpp file and deleted main.H